am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -31,23 +36,40 @@ void setupTest() {
     }
 
     @Test
-    void filterResource_whenRowExistWithAccessorIdAndResourceId_ReturnPassedJsonObject() {
-        ams.createResourceAccess(createRecord(resourceId, ACCESSOR_ID, EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS));
+    void whenAddTwoRecordsWithSameResourceIdAndUserIdButDifferentAttribute_ReturnsTwoAttributes() {
+        ams.createResourceAccess(
+            createRecord(resourceId, ACCESSOR_ID, EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS, "/test"));
+        ams.createResourceAccess(
+            createRecord(resourceId, ACCESSOR_ID, EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS, "/name"));
 
         FilterResourceResponse result = ams.filterResource(ACCESSOR_ID, resourceId, DATA);
 
+        assertThat(result).toString().contains("test");
```

Comment:
I beg you, do not use `toString` method in tests!!! I am not even sure what that tests is checking. A word `test` can appear in many places of `FilterResourceResponse`.

Looking for analogy if I was to have test that assures me that Becky is Becky I would not write it as `becky.has("green").has("long")` because these assertion could describe for example green eye or green scarf or something.

Reply:
👍